src/mumble/ListenerLocalVolumeSlider.cpp

```diff
@@ -10,12 +10,16 @@
 
 
 ListenerLocalVolumeSlider::ListenerLocalVolumeSlider(QWidget *parent)
-	: VolumeSliderWidgetAction(parent) {}
+	: VolumeSliderWidgetAction(parent) {
+	this->blockSliderSignals(true);
```

Comment:
We don't use the `this` keyword where it can be avoided (I don't like that either but the coding guidelines are what they are).

Also it'd be easier if you were to call this function in the super class's constructor instead of having to do it explicitly in both sub-classes